### ISSUE TITLE
The space left behind after a typing notification can look a little bit like a glitch

### ISSUE THREAD

**Issue by lampholder** [vector-im/riot-web#8552]
Rick observed that, if the typing notif is brief (or you don't notice it) it kinda looks like an unexplained gap.

**Comment by ShadowJonathan** [comment]
This actually has real impact for me, copy-paste from #19758;

> This could lead into https://github.com/vector-im/element-web/issues/16304, but i'm mainly focusing on the problem case here, which is that the "typing" notification has a noticeable detrimental effect with me on observing the chat.
> 
> Call it "mental muscle memory", but personally, i've found myself always equating "timeline moves up" (in another window) to "new message", *explicitly* when there's something new to read.
> 
> This makes element hard to keep open in another window, as the timeline is moved on typing notifications, but more often than not *isnt* (substantially) moved when the message actually comes out. This makes me unconsciously anxious of missing out messages, when i've (multiple times) instantly looked over to element once someone started typing (as the timeline moved in my peripheral vision), became frustrated when its just someone typing, and when looking at the timeline later, saw that person has posted a message after all, which i didn't notice.
> 
> There's two key inconsistencies here with other messages;
> 1. "is typing" banners move up the timeline
> 2. commonly, a message posted after an "is typing" banner will not move the timeline substantially, as it is already "raised up"
> 
> 1 triggers me looking over, while 2 doesn't, this makes my experience with element sub-par.
> 
> Even if 2 wasn't the case, 1 "blurs" the meaning of "timeline moving up" enough so that i'm not inclined to look over. This causes some severe anxiety.
> 
> This problem exists on both element web and element ios (i can't confirm if it also applies to android)

**Comment by ShadowJonathan** [comment]
I retracted my comment, i realise that i accidentally hijacked this issue. This issue is about a regression, and #19758 is about an UX grievance.

**Comment by scotttrinh** [comment]
Not sure if this is a recent regression, but was discussing this issue with @ShadowJonathan on a matrix server and was encouraged to post a screen recording of a possibly related issue I was seeing

https://user-images.githubusercontent.com/1682194/142932800-f90f4ab7-525b-45af-b680-e2753cd8474e.mov

I'm on version 1.9.3 on macOS installed from nix.

**Comment by ShadowJonathan** [comment]
(I just deleted and reverted some comments, sorry about that, i directed @scotttrinh to this issue out of the presumption that this issue was more related to theirs, but this issue is related to the gap left behind by the typing indicator, while #19855, the issue i broke it out with, is related to the regression with the timeline jumping up and down)

**Comment by kgiebeler-xq-tec** [comment]
> TLDR; This messes with me being able to follow chat properly when on a second monitor.

Same for me on a bigger screen. I often have to minimize Element because that no-op-scrolling is very distracting. On top of that I often miss the actual submission of a post because the is no additional _scroll event_ indicating the author is done writing - the space of the typing indicator is just being re-used by the actual message.

**Comment by weirdreality** [comment]
Today is the 7 year anniversary of this (good job to Rick for observing it), what exactly is the progress on it getting fixed?
